draft-ietf-quic-multipath.md

```diff
@@ -566,12 +561,13 @@ When only one path is available, servers MUST follow the specifications
 in {{QUIC-TRANSPORT}}.
 
 When more than one path is available, hosts shall monitor the arrival of
-non-probing packets and the acknowledgements for the packets sent over each
-path. Hosts SHOULD stop sending traffic on a path if for at least the period of the
+packets and acknowledgements for packets sent over each
+path. Hosts MAY consider closing a path if for at least the period of the
```

Comment:
Note, this is a normative change. There is no real reason to close a path after a time out as you may just use other paths currently. However, there might be good reasons to keep it open for potential later use (e.g. if the path is a standby path). Therefore we should not have a SHOULD here. I also opened issue #397 for more discussion.

Reply:
That behavior is defined in the section {{idle-time-close}}. Do we need to define it in two places?

Reply:
This is the section on idle timeout!

Reply:
Oops. Yes, I like "MAY".